workflows/promote-nightly.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Promotes a nightly release.
+ */
+
+import {createVersionsUpdatePullRequest, runPromoteJob} from './promote-job';
+
+const jobName = 'promote-nightly.js';
+const {AMP_VERSION} = process.env;
+
+void runPromoteJob(jobName, async () => {
+  if (!AMP_VERSION) {
+    throw new Error('Environment variable AMP_VERSION is missing');
+  }
+
+  await createVersionsUpdatePullRequest(() => ({
+    versionsChanges: {nightly: `04${AMP_VERSION}`},
```

Comment:
Do we need the `04` prefix or can we do a replace on the last 13? It would help in not having to worry about the prefixes for all the channels.

Reply:
We have the JSON verifier step in the CI, and these numbers aren't going to change so it's an easy explicit solution. There _could_ be a situation where we disable nightly temporarily (manually setting it to `null`) in which case a S&R won't work

Reply:
It would be simpler if the prefix weren't carried over. The json schema CI check is perfect in catching bad S&Rs.

Do you see disabling nightly as a likely scenario? It would fail the same CI check anyways. And an null check in the code is cleaner to me than having prefixes in places other than `versions.json`